portfolio/src/main/java/com/google/sps/data/BookReader.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+import com.google.sps.data.Book;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.ServletException;
+
+/**
+ * Class that takes in a CSV FileName as the constructor parameter
+ * and populates a list of Books with title, genre, and reviews.
+ */
+public class BookReader {
+  String path;
+  public BookReader(String path) {
+    this.path = path;
+  }
+
+  public ArrayList<Book> makeBookList() throws ServletException{
+    ArrayList<Book> listOfBooks = new ArrayList<Book>();
```

Comment:
You could use ImmutableList (using builder) here:
https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.Builder.html

It is preferable to return immutable types.